### PR TITLE
Padding changed to avoid clipping on color swatches

### PR DESCRIPTION
**Pull request by DivyaPrabhakar**
References issue #3549  

- [x] @placegraphichere to sign off on the design

### CODE REVIEW

File: src/style/panel.less [hunk 1]
```diff
@@ -133,12 +133,11 @@
 }
 
 .formline__no-padding {
-    padding-top: 0.2rem;
     padding-bottom: 0.1rem;    
 }
 
 .formline:first-child {
-    padding-top: 0;
+    padding-top: 0rem;
```
Comment (iwehrman) [inline src/style/panel.less]: I don't think this is valid CSS? It probably works either way, but I was under the impression that 0 should always be unit-less.

**Comment by placegraphichere** [comment]
@iwehrman units should not matter, but we should remove it. @DivyaPrabhakar I will review

**Comment by iwehrman** [comment]
I think this is referencing the wrong issue? #3459 was about the scripting time needed for document activation.

**Comment by DivyaPrabhakar** [comment]
@placegraphichere I have removed the units so back to you. 
@iwehrman You are right, it references issue #3549. I have changed it.

**Pull request merged**